sips/sip-102.md

```diff
@@ -0,0 +1,142 @@
+
+Integrate with Optimism to enable the staking of `SNX` on L2 - an environment with faster transaction throughput and much cheaper gas costs.
+
+## Motivation
+
+<!--This is the problem statement. This is the *why* of the SIP. It should clearly explain *why* the current state of the protocol is inadequate.  It is critical that you explain *why* the change is needed, if the SIP proposes changing how something is calculated, you must address *why* the current calculation is innaccurate or wrong. This is not the place to describe how the SIP will address the issue!-->
+
+As most users are aware, Ethereum has a major issue with scalability. The recent DeFi summer of 2020 caused gas prices to surge, pricing out a large number of holders who could not justify spending so much of their Ether to transact. Eth 2.0 is coming, and with it, [a rollup-centric approach to scaling](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698):
+
+> the Ethereum ecosystem is likely to be all-in on rollups (plus some plasma and channels) as a scaling strategy for the near and mid-term future. - Vitalik Buterin
+
+After investigating the various options, and considering the trajectory of Eth 2.0, Optimism was deemed to be the best fit for Synthetix's needs. In short, Optimism's Virtual Machine (the OVM) allows contract code from L1 to be ported to L2 wholesale, with only minor modifications. For more details on the choice of L2 provider, please see the blog post [Why Optimism?](https://blog.synthetix.io/why-optimism/)
+
+In order to tease out any challenges integrating with a new layer 2 technology, a [testnet trial](https://blog.synthetix.io/optimistic-ethereum-l2-testnet/) has been underway on the Goerli testnet since late September. This trial has tested connecting a custom L2 deployment of Synthetix to one on Goerli, and supported deposits, withdrawals and staking. After a couple of months working closely with the Optimism team, the code is ready to be released on mainnet.
+
+## Specification
+
+> Note: This SIP proposes using L2 for staking only. Future proposals may well introduce support for exchanging as well.
+
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+
+<!--This is a high level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
+
+The creation of two new contracts - a deposit contract for L1 and withdrawal for L2.
+
+1. A bridge contract on L1: `SynthetixBridgeToOptimism` that supports the following:
+
+   - When invoked by a user for a deposit of SNX, will hold the SNX as a deposit, will burn any escrowed L1 SNX and send a message to L2 representing the deposited SNX and escrow entries
```

Comment:
This may actually need to be split up in 2 functions depending on the size of the escrow entries to migrate.

Reply:
Noted, but probably too specific to include here in the SIP